src/clients/ntp-client_proposal.rb

```diff
@@ -387,27 +376,13 @@ def Write(param)
       if NetworkService.isNetworkRunning
         # Only if network is running try to synchronize the ntp server
         Popup.ShowFeedback("", _("Synchronizing with NTP server..."))
-
-        Builtins.y2milestone("Running sntp to sync with %1", ntp_server)
-
-        # -S: do set the system time
-        # -t 5: timeout of 5 seconds
-        # -K /dev/null: use /dev/null as KoD history file (if not specified,
-        #               /var/db/ntp-kod will be used and it doesn't exist)
-        # -l <file>: log to a file to not mess text mode installation
-        # -c: causes all IP addresses to which ntp_server resolves to be queried in parallel
-        ret = SCR.Execute(
-          path(".target.bash"),
-          "/usr/sbin/sntp -S -K /dev/null -l /var/log/YaST2/sntp.log " \
-          "-t 5 -c '#{String.Quote(ntp_server)}'"
-        )
-        Builtins.y2milestone("'sntp %1' returned %2", ntp_server, ret)
+        NtpClient.sync_once(ntp_server)
         Popup.ClearFeedback
       end
 
       return :ntpdate_failed if ret != 0
 
-      # User wants to more than running sntp (synchronize on boot)
+      # User wants to more than running one time sync (synchronize on boot)
```

Comment:
I'd use ```wants more than``` instead of ```wants to more```

Reply:
right, I just change second part of sentence, but first part is wrong from begging. Will fix it.